Automattic/jetpack
# Contact Form: align checkbox and radio button baselines

Continuation of https://github.com/Automattic/jetpack/pull/34272.

## Proposed changes:

This PR aligns the Contact Form radio buttons and checkboxes with their labels' baseline, as shown in the capture below.


<img width="500" alt="Screenshot 2023-12-01 at 2 39 32 PM" src="https://github.com/Automattic/jetpack/assets/1620183/cc0f664a-aec4-4ecc-9004-07ed0b805d70">

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a test site with this branch
- Create a new post
- Add the Contact Form block to the page
- Add a Multiple Choice and Single Choice fields
- For each field, add an option with a long text
- Open the post preview
- Notice fields look like they do in the capture above
- Back in the editor, increase the font size
<img width="289" alt="Screenshot 2023-12-01 at 2 39 58 PM" src="https://github.com/Automattic/jetpack/assets/1620183/71ebf031-cd4c-4a05-adc6-567bf894b073">

- Check that the checkboxes and radio buttons in the preview match the new font size
<img width="500" alt="Screenshot 2023-12-01 at 2 40 21 PM" src="https://github.com/Automattic/jetpack/assets/1620183/b2656f53-5a37-47ce-b98c-389619e13311">
<img width="500" alt="Screenshot 2023-12-01 at 2 40 31 PM" src="https://github.com/Automattic/jetpack/assets/1620183/5645971a-9149-4442-96dc-7174615dc1d0">

- Optionally, test the PR with the _Outlined_ and _Animated_ styles
